src/pages/SaladsPage.jsx

```diff
@@ -1,7 +1,70 @@
 import BaseLayout from "../layout/BaseLayout";
 
+import _1 from "../assets/salads/apple-salad5-resized.jpg";
+import _2 from "../assets/salads/avocado-rocket2-resized.jpg";
+import _3 from "../assets/salads/carrot-salad-2-resized.jpg";
+import _4 from "../assets/salads/cous-cous-with-summer-greens1-resized.webp";
+import _5 from "../assets/salads/noodle-salad1-resized.webp";
+import _6 from "../assets/salads/potato-salad-1-resized.jpg";
+import _7 from "../assets/salads/pumpkin-fetta-and-pine-nuts-1-resized.jpg";
+import _8 from "../assets/salads/rice-2-resized.webp";
+import _9 from "../assets/salads/roasted-cauliflower1.webp";
+import _10 from "../assets/salads/sweet-chilli-dressing1-square-resized-e1522656058614.jpg";
+import _11 from "../assets/salads/tomato-mezze2.webp";
+import _12 from "../assets/salads/zucchini-3-resized.jpg";
+
 function SaladsPage() {
-  return <BaseLayout>salads</BaseLayout>;
+  return (
+    <BaseLayout>
+      <div className="salads-images">
+        <p>
+          A selection of some of my favourite salad recipes. Wonderful all year
+          round, but especially beautiful in spring and summer, when the weather
+          calls us outdoors. Recipe links below, or see all my salad recipes.
+        </p>
+
```

Comment:
In the future, `alt` should be a string that describes the image

Reply:
noted sir.